### PR TITLE
cleanup args if

### PR DESCRIPTION
**Pull request by stoeffel**
@przemcio I tried to cleanup the `if` in `wrappedTask` how do you like it?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,21 +11,21 @@ module.exports = function (gulp, processArgv, callbackName) {
...
-    if (!taskDefinition && R.is(Function, taskDependencies)) {
-      taskDefinition = taskDependencies;
-      taskDependencies = undefined;
-    }
-    taskDefinition = taskDefinition || function () {
-      };
+    var task = R.ifElse(
+      R.is(Function),
+      R.identity,
+      R.always(R.or(taskDefinition, noop))
```
Comment (stoeffel) [inline index.js]: IMO this could be without the or.
```js
R.always(taskDefinition)
```

**Comment by przemcio** [comment]
Hi sorry I have one week off ;) I will check it today evening

**Comment by stoeffel** [comment]
Hi. No need to apologize:-) I hope you enjoyed your week off.

**Comment by przemcio** [comment]
The version with R.always(taskDefinition) will not work If taskDefinition is undefined. I have added one additional test and I did inline refactoring of noop function.

**Comment by stoeffel** [comment]
LGTM. had the same but removed tge or :-)

**Pull request merged**